releasenotes/notes/linear-inequality-to-penalty-70b5277b8edda012.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Introduced a new converter class ``LinearInequalityToPenalty`` for ``QuadraticProgram``.
+    It converts some inequality constraints to penalty terms without adding slack variables.
```

Comment:
I note this says `some inequality constraints` so some presumably some are not supported. Is it possible to say anything more here to qualify say the scope/aspect of those that can be converted so the reader has some idea which?

Reply:
Is this acceptable ?
---
Introduced a new converter class ``LinearInequalityToPenalty`` for ``QuadraticProgram``.
It converts known inequality constraints to penalty terms without adding slack variables.
The known inequality constraints are described in this paper: 
   A Tutorial on Formulating and Using QUBO Models, arXiv:1811.11538 <https://arxiv.org/abs/1811.11538>,
   Section 4: Creating QUBO Models Using Known Penalties"
In addition to the known constraints, this converter converts more general form. Please refer to
API documentation for full list of supported inequality constraints.
----

Reply:
It is better to describe the actual patterns. I will rewrite that part.

Reply:
I included the patterns in reno and docstring of `convert`.  4a479f3